sunkit_instruments/response/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+A subpackage for computing instrument responses
+"""
+
+from sunkit_instruments.response.thermal import *
```

Comment:
Nitpick, can we add an all and explicit imports?

Reply:
Done